### PR TITLE
Allow admins to view issues and notes

### PR DESCRIPTION
**Pull request by urbanwide**
See [trello card](https://trello.com/c/Ss7zfZPh/2482-allow-admins-to-view-issues-and-notes) for further info

### CODE REVIEW

File: app/views/admin/schools/notes/index.html.erb [hunk 1]
```diff
@@ -1,34 +1,19 @@
...
-  </div>
-</div>
+<%= render 'header', title: "#{@school.name} Notes & Issues" do %>
+  <%= header_nav_link 'School group', admin_school_group_url(@school.school_group) if @school.school_group %>
+<% end %>
+
 <p>
-  <%= link_to 'New Note', new_admin_school_note_path(@school), class: 'btn btn-primary' %>
-  <%= link_to 'New Issue', new_admin_school_note_path(@school, note_type: :issue), class: 'btn btn-primary' %>
+  <%= link_to "#{((fa_icon('sticky-note text-secondary')))} New Note".html_safe, new_admin_school_note_path(@school, note_type: :note), class: 'btn btn-primary btn-sm' %>
```
Comment (jhigman) [inline app/views/admin/schools/notes/index.html.erb:6]: Are the multiple brackets around the fa_icon required?

File: app/views/admin/schools/notes/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<%= render 'header', title: "View #{@note.note_type.capitalize} for #{@school.name}" do %>
+  <%= header_nav_link 'All school notes & issues', admin_school_notes_url(@school) %>
+<% end %>
+
+<%= render 'note', note: @note, index: false %>
```
Comment (jhigman) [inline app/views/admin/schools/notes/show.html.erb:5]: Newline missing?

File: app/views/admin/schools/notes/index.html.erb (hunk 1 above)
Comment (urbanwide) [reply app/views/admin/schools/notes/index.html.erb:6]: Hmm no, not sure how that happened. Thanks for picking that up!

File: app/views/admin/schools/notes/show.html.erb (hunk 2 above)
Comment (urbanwide) [reply app/views/admin/schools/notes/show.html.erb]: Thank you! I'll add all these in 👍

**Review by jhigman — APPROVED** [review]

**Pull request merged**